### ISSUE TITLE
The stun meta is kinda boring

### ISSUE THREAD

**Issue by QuietlyWhisper** [space-wizards/space-station-14#18332]
## Description

(Taken from the document)
Basically in the current meta stuns are the best weapon. With just batons alone, two hits can permanently remove a player or antagonist from the game with little to no effort. It’s likely one of if not the most boring combat types as well, due to no actual physical damage being done, making armor and weapons redundant.

I've attached a proposed changes google doc with open comments so anyone can suggest changes or issues.

Nobody is working on this yet, but it might be a good start.

https://docs.google.com/document/d/1t36eXz70i6BuLwGCpucSBfiRh8t7lzRaJJwylTLJ-nI/edit?usp=sharing

**Comment by deltanedas** [comment]
stun resistant chems dont exist
stims and meth just reduce the knockdown time thats it

**Comment by Emisse** [comment]
stuns are good

**Comment by HoidC** [comment]
> stuns are good

Have you read the document

**Comment by asperger-sind** [comment]
> stuns are good

tl;dr, stuns are meh, stuns have always been ass to balance, play with or against, stam damage made it a bit better but didnt fix them. i like killing hardstuns and like softstuns because its better game design and allows for more counterplay and cool stuff, but im concerned that stuns may get overnerfed and become bad.
example of current stuns: secoff engages antag, antag gets batonned twice and fucking dies with little counterplay, sec is rewarded too much for clicking on a guy twice.
example of proposed stuns: secoff engages antag, antag gets batonned twice, antag then retaliates and tries to shoot the secoff and fuck off, sec is rewarded for having a good guess on an antag but not rewarded so much as to get a free arrest
i hope proposed stuns get merged n shit

Stuns aren't good, they're okay at best. They're not as fun to use as non-stun options and annoying as fuck to play against while causing a lot of balance problems and have been like that for as long as I can remember, stamina damage made it slightly better with tasers ceasing to exist until that one PR that reintroduced them but aside from that, gameplay-wise, stuns haven't changed much.

I like the proposal of this document to kill hardstuns and replace them with softstuns, this allows for more counterplay and should make stuns less of a dumpster fire design-wise, speaking of, this design doc exclusively nerfs stuns with nothing to counterbalance them, this is a problem because this might result in stuns getting overnerfed and security ending up resorting to lethals 9/10 times because stuns aren't usable.

I think this is good and demonstrate this example in the current stun iteration:

Security has placed a warrant on an antag, the antag is unaware and has thus far not done anything that they think security noticed, the antag has been confronted by an officer in a somewhat isolated area (hop booth on packed for example), the suspect instantly gets stunsticked without the officer saying a word, after the first hit they cannot run away and have about a window of about a second before they are hit the second time and are stunned for the second time before being ultimately cuffed.

Now then, security played this one well on a gameplay scale, they outed an antag and put them in an unfavorable position, they should be rewarded for this, but the current reward is disproportionally large for what security has done.

The antag, of course, can still attempt to counterplay this, they can slip the officer, attempt to stunbaton them if they have a stunbaton of their own, drink or inject a stun shortener (e.g ephedrine, stims, etc), try to blast the officer with a shotgun, roll for a disarm, etc.

These are the majority of the strong options you have for counterplay provided you're in this situation, most of them aren't guaranteed to work and require you to have specific gear (not everyone walks around with shotguns).
In general, counterplay options for this matter are weak and unreliable, disproportionally so for the effort the secoff has put into place to stunstick you twice.

Let's assume we have the iteration of stuns from the design doc above:

Same scenario, secoff engages the suspect, batons them twice, the suspect is now facing the secoff in a one-on-one engagement, they can for example move towards maints while hitting them with antag weaponry or fighting back using a gun, the secoff while being rewarded with an upper hand in the fight due to pre-emptively figuring out who the antag is, still has to fight the antag in order to arrest them.

This scenario not only has more counterplay while rewarding the secoff and potentially rewarding the antag for playing well and making a more interesting, multilayered, less predictable and deep combat interaction for both parties involved, potentially allowing for other parties (other secoffs, antags, random people, etc.) to join in the interaction as well.

I personally prefer scenario №2 over №1 due to the reasons stated above, and hope that stuns get a rework to stop being a dumpster fire in a game design sense.

**Comment by deltanedas** [comment]
average stunmeta enjoyer

**Comment by HoidC** [comment]
> 

uhm just dont approach target and always kill at range??!

**Comment by graevy** [comment]
i feel that melee range has always been anything-goes, and these are good changes for h/mrp. however, h/mrp requires non-damaging takedowns for space law, and asimov secborgs would need to be addressed. this also means that ordinary interdictions would be more likely to escalate, leading to more bwoinks, but maybe making medbay funner.

i agree that stuns aren't very interactive gameplay. i think we need to invent the "Other means of takedowns" in the document to continue having this discussion?

**Comment by HoidC** [comment]
> i feel that melee range has always been anything-goes, and these are good changes for h/mrp (because lrp etiquette avoids melee range). however, h/mrp requires non-damaging takedowns for space law, and asimov secborgs would need to be addressed (maybe they continue to have good stuns?). this also means that ordinary interdictions would be more likely to escalate, leading to more bwoinks, but maybe making medbay funner.
> 
> i agree that stuns aren't very interactive gameplay. i think we need to invent the "Other means of takedowns" in the document to continue having this discussion?

the takedown is non lethal

they're hugely debuffed movement speed wise, they drop what they're holding and unless the arestee escalates sec can take them in peacefully

**Comment by QuietlyWhisper** [comment]
I've moved everything to the github.

**Comment by graevy** [comment]
> they're hugely debuffed movement speed wise, they drop what they're holding and unless the arestee escalates sec can take them in peacefully

if the combat flow changes from
- stun -> cuff/kill
- stun -> maybe be attacked -> cuff/kill

either we proceed as before, or two players with stuns stalemate until:
1. someone uses lethals,
2. ammo exhausts, or
3. third party intervenes

so
- lethals are incentivized. this is a mixed bag, but does seem more fun
- movement is discarded for turn-based lethal-spamming while maintaining a stun debuff (if i'm understanding this correctly)
- outside interference is more impactful, and individuals are worse against crowds

i think players complain about the stun system because they want more nuance than just the neutral phase of combat.

i think ss13 "combat" is best when players are reading body language, positioning themselves, etc. and so i think i would look there first to change the stun system. what if batons had a short activation delay after being equipped every time? what if stunning projectiles moved slower? etc.

**Comment by asperger-sind** [comment]
> what if batons had a short activation delay after being equipped every time? what if stunning projectiles moved slower? etc.

That wouldn't solve stuns being poorly designed at their core, people would just take out and activate stunsticks 2 seconds before they engage and nothing else changes, sure you have slightly more time to look at them and react, but it doesn't fix stuns.

Making stun projectiles slower doesn't fix anything either, now people'll just matrix dodge disablers and rubbers more often than not, which they do already.

**Comment by asperger-sind** [comment]
> I feel that melee range has always been anything-goes, and these are good changes for h/mrp (because lrp etiquette avoids melee range)

"LRP etiquette" does not avoid melee range, in fact, more often than not melee is a more common method of combat rather than ranged weaponry not only by sample size, there are good builds for melee combat if you're playing antag, and melee is reliable and easy to obtain as non-antag (makeshift weaponry, stuff like circular saws or spades if your job has access to them)
Security currently has stunsticks, which are by far the best melee weapon in the game right now for most combat scenarios.

**Comment by DerbyX** [comment]
**[1]** This system doesn't work with how tasers should function.
**[2]** In general the stun mechanics are fine in my opinion. Yeah no one likes being hit twice and cuffed but if you know you're doing bad things and don't take preventative measures against getting arrested then who's really at fault, the system? No.
**[3]** The stamina system should be more integrated with fighting instead of a whole stun system rework. For example if you take multiple shots from beanbags you become staminacrit able to "crawl" but if you're wacked with a stunbaton it lowers your stamina and puts you into stuncrit(Unable to act). Tasers should stuncrit as well, it's a taser.
**[4]** Preventative measures such as energy drinks(legal) or drugs(illegal) can boost stamina temporarily negating the loss during fights or allowing the perp to get away.
**[5]** Other preventative measures already exist like the EMP (Both implant and not) and the Freedom Implant. You can use chimera gear, voice changer, hide in maints, or even swap your body. Read [2].

Results:
• Tasers still function like tasers (They're T3 for a reason)
• The stamina system now having a purpose and gives meaning to it.
• Stun batons are still useful but new ways of countering them can be done either through simple (Energy Drinks/Coffee) or extreme(Drugs/Implants) means.
• Criminals can still be caught with relative ease if they don't put any effort into taking precautionary measures.
• Criminals won't be gunned down the second they start resisting.

Addendum:
"**Security will be required to use weapons vs nuclear operatives**" was never an issue if nukies stayed together or communicated well enough to not get arrested.

"**Crew and syndicates can resist arrest violently, in which case security may need to use lethal force to put criminals in crit.**" I'm all for lethals but no one should be gunned down if they don't actively try to kill someone(My system prevents this) or are impossible to arrest normally.

"**Current server rules already allow for this.**" isn't a good reason.

"**Other means of takedowns will be explored as stuns will not always be the best solution to every problem anymore.**" they've always been explored but most situations call for less lethal options. And there's only two "means". Lethal and less lethal. Read the second addendum.

"**People may violently resist arrest at their own risk. The officer is not at fault for raising escalation if you attack them to prevent being cuffed. Officers have already been given equipment if this situation occurs.**" Violently resisting is already a thing, it's called running away and using violent means to escape (guns, explosions, etc). Again look at the second addendum. Officers "having equipment if this situation occurs" isn't a good reason either.

"**More gunfights and less melee fights?**" is an RP/situation issue not a stun system issue.

These are my general thoughts, feel free to not care about them or make note of it.

**Comment by Aisu9** [comment]
melee combats using stamina as a resource so you have to use your muscles AND brain and not just going head first with no drawbacks. (attacking require stamina, weapon dealing stamina damage, heavy attack using and dealing more stamina damage)

**Comment by Equivocateur** [comment]
> In general the stun mechanics are fine in my opinion. Yeah no one likes being hit twice and cuffed but if you know you're doing bad things and don't take preventative measures against getting arrested then who's really at fault, the system? No.
> Other preventative measures already exist like the EMP (Both implant and not) and the Freedom Implant. You can use chimera gear, voice changer, hide in maints, or even swap your body.

**The only** counterplay in the game currently is the emp implant/grenade. Using it is a mark for death if the seccie being emp'ed squeals about the fact that its in play. This invites using guns since you invalidate the reason to use nonlethals. Chameleons are impractical in operation. A cham kit and a voice mask will set you back in excess of 10 TC, adding with the EMP implant of 5 TC you are left with fuckall choices to play with. 
Freedom implant is also a mark for death and you will be gunned down instead of stunbatoned.

> For example if you take multiple shots from beanbags you become staminacrit 

How to fuck over bar and make security even more oppresive without any chance to fight back.

> Tasers should stuncrit as well, it's a taser.
> Tasers still function like tasers (They're T3 for a reason)

Taser was an admin item and I have no idea why just cutting the amount of charges in it was sufficient - its a perpetual stun machine that can infinitely stamcrit with no counterplay even with a team (they also get to eat the taser). Just because its T3 changes nothing as a good sci team can consistently research 2 full disciplines in any given round.

> Stun batons are still useful but new ways of countering them can be done either through simple (Energy Drinks/Coffee) or extreme(Drugs/Implants) means.

Desoxyephedrine currently gives enough time for sec to cuff you if they know what youre doing, at best you narrowly escape the do-after of getting cuffed. Desoxyephedrine takes a lot of prep work to make for little payoff. Stims got nerfed into literal shit because of local murderbone killsquads annihilating entire stations. 8 TC for 60 seconds is not worth it. Stimkits give even less value.

> "Crew and syndicates can resist arrest violently, in which case security may need to use lethal force to put criminals in crit."
> "People may violently resist arrest at their own risk. The officer is not at fault for raising escalation if you attack them to prevent being cuffed. Officers have already been given equipment if this situation occurs."
> "Current server rules already allow for this." isn't a good reason.

The server rules forbid using lethals the moment you get punched as security per rule 20. Furthermore it was already discussed that punches wont get you out of a security officer's hold - you'd need something harder-hitting to break it.

The point of this issue is to:
A) Remove the stunbaton from the pedestal of **the** best weapon in the fucking game in a game where the map design is incredibly claustrophobic and forces close quarters combat.
B) Make having a literal gun in your pocket a counter to a security officer jumping you from a corner with a baton without relying on ~~wallhacks~~ reflexes to click the hotbar key (emp implant) or gambling you wont be killed the moment you snap your cuffs (freedom implant)

**Comment by Interrobang01** [comment]
an easy way to balance stuns would be to make stuns like lethals but without being lethal

- add a new damage type, Stun Damage 
- have it automatically regenerate at a fast rate
- crits induced with this new damage type do not result in airloss or vision obstruction
- make all the stun weapons deal this type of damage 
- one or more other things to fix that problem you just thought of

if it's still unbalanced after that then we have bigger problems



...also don't disablers exist? everyone's talking about melee but aren't disablers also effective?

**Comment by metalgearsloth** [comment]
> 
    add a new damage type, Stun Damage
    have it automatically regenerate at a fast rate
    crits induced with this new damage type do not result in airloss or vision obstruction
    make all the stun weapons deal this type of damage
    one or more other things to fix that problem you just thought of

How is this different to our current system?

**Comment by Emisse** [comment]
that is literally our current stun system lmao

**Comment by Interrobang01** [comment]
i'm suddenly aware of that now

**Comment by metalgearsloth** [comment]
As far as I can discern it just seems like we need to make tweak to stamina values, potentially 3-4 hits baton with movement penalty after 1-2 and bumping disabler.

Addressing whisper:

> Stuncritee falls over, dropping anything they were holding.

This should already happen.

> Stuncritee has a -% movement penalty, meaning they can move while stamina crit, although very slowly. This would be called crawling.

This already happens at a threshold I don't recall.

> Stuncritee is able to use hands and implants while in stun, picking up items dropped on the floor, and is able to attack.

Being able to do interactions and move should not be able to happen as then it's no longer a stun it's a slow which is useless against someone who has an esword / syndie fireaxe and is in melee range.

> This should resolve any issues of nuclear operative equipment being easily made obsolete.

Or tweak stamina values if we really want to, but keep in mind most sec won't have lethals aside from their sidearm when nukies arrive as most of the time armory is hit first so sec become significantly weaker.

> Holding a stuncritee stops movement. This can be reused from the cuffed code.

Except they can now attack you with a weapon and mess with you to interrupt it?

> Resolves any issues of players moving out of cuff range or interrupting this by moving.

Leaving the current stun as is also solves that because they can do neither and also can't interrupt you.

> Cuffing should have a higher damage threshold to cancel, so punches may not cancel it but someone with a weapon could.

This would mean you can cuff someone who has slipped which seems silly (as mentioned above they can still crawl hence movement no longer breaks it) and now a slip-meta forms where throwing soap and cuffing is significantly better.

> Resolves a handful of issues where security wouldn’t be able to arrest minor crimes.
> Slips, including being stunned via pie should also fall under this effect.

**Comment by QuietlyWhisper** [comment]
> As far as I can discern it just seems like we need to make tweak to stamina values, potentially 3-4 hits baton with movement penalty after 1-2 and bumping disabler.

Probably for the best

**Comment by asperger-sind** [comment]
> This would mean you can cuff someone who has slipped which seems silly (as mentioned above they can still crawl hence movement no longer breaks it) and now a slip-meta forms where throwing soap and cuffing is significantly better.

We already have slip meta (faster, doesn't need charge, stuns from one hit instead of 2, basically instant, for the worthy, less storage space required) and removing it would be even sillier than having it :troll:

**Comment by asperger-sind** [comment]
oh my god why does it use the discord emoji in the relay oh no

**Comment by Equivocateur** [comment]
> This would mean you can cuff someone who has slipped which seems silly 

This is already a thing. Slipstun with normal soap (which also completely paralyzes you for some reason) gives enough time to cuff you.

**Comment by Partmedia** [comment]
While I agree there are problems with the stun meta, I'm also concerned about security having to escalate to lethals too much. The stun baton already runs out rather quickly, and as you mention requires at least two hits. And unless you have some handcuff key bindings handy, there is not a lot of time to cuff someone and you end up having to re-stun them again.

**Comment by JoeHammad1844** [comment]
> uhm just dont approach target and always kill at range??!

Due to the nature of the game, being in a space station with somewhat narrow corridors this just isn't possible at least half the time

**Comment by deltanedas** [comment]
@asperger-sind dont :troll: do `:trollface:` :trollface:

**Comment by ashkitten** [comment]
it strikes me as a bit odd that stun/stamina is effectively unrelated to overall damage. i think it would make sense to have other damage types affect maximum stamina, so it's easier to stun someone who's near crit than a healthy uninjured person. this would also make it less annoying to slip on soap or lube if someone just leaves it out, you don't get stunned for the full three seconds.

**Comment by kontakt** [comment]
Stale. Balance and feature discussions should be brought to the relevant workgroup.
